src/xdocs/config_blocks.xml

```diff
@@ -173,15 +174,16 @@ switch (a)
         <p> Configuration: </p>
         <source>
 &lt;module name=&quot;EmptyBlock&quot;&gt;
-    &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_DEFAULT&quot;/&gt;
+  &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_DEFAULT&quot;/&gt;
 &lt;/module&gt;
         </source>
         <p> Result: </p>
         <source>
-              switch (a) {
-              default:        // no violation for "default:" as empty block belong to "case 1:"
-              case 1: { }
-              }
+switch (a)
+{
+    default:        // no violation for "default:" as empty block belong to "case 1:"
```

Comment:
please return "{" to the end of line , this Check does not validate this, so it is better to keep example as compact.
please make indentation as 2, here is 4.
Please make one space after ":"

Reply:
@romani I changed the '{' after the switch statement by considering the original source code on line 66. So does that also needs to be changed considering this?

```
switch (a)
{
    case 0:
        // Never OK, break outside block
```

Reply:
> I changed the '{' after the switch statement by considering the original source code on line 66. So does that also needs to be changed considering this?

"{" on new line is very old style, so this example was added in time when it was wide used.
Checkstyle in general can enforce multiple code styles, so our sample files will never be written in same style, to show user code with violations and what is expected.
I do not recommend you to change style of code from sample, as it could change behavior of Check and sample might not be valid anymore.
Lets change indentation only for now.

> Also does the indentation for Java source code should remain to 2 similar to XML?

yes, spacing is not affecting any Check , except for http://checkstyle.sourceforge.net/config_misc.html#Indentation . Non of this sample should be changed, as amount of spaces in this Check is part of validation.

Reply:
Ok, thanks for the clarification. Just to reiterate my points of action
 - I would revert back the "{" at the places where I have changed it to a new line
 - I would change the indent space to 2 under source blocks

Reply:
@romani  Done